src/Mailtrap.php

```diff
@@ -72,23 +70,32 @@ public function _initialize()
      *
      * @param \Codeception\TestCase $test
      */
+
+    /*
     public function _after(\Codeception\TestCase $test)
     {
-        if ($this->config['cleanup']) {
-            $this->cleanInbox();
-        }
+        $this->cleanInbox();
     }
+    */
```

Comment:
why?

Reply:
Agreed with @sergeyklay, the cleanup functionality is useful when running test in some cases, you can always deactivate it from the config.